core-primitives/node-api/metadata/src/pallet_proxy.rs

```diff
@@ -0,0 +1,39 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{error::Result, NodeMetadata};
+
+/// Pallet name:
+const PROXY: &str = "Proxy";
+/// the deposit needed to register up to 20 proxies in native parentchain token
+pub const PROXY_DEPOSIT: u128 = 21_000_000_000_000;
```

Comment:
not sure this is the right place for such a constant. We may be able to derive this amount from metadata but I couldn't bother

Reply:
in the interest of swift demo delivery, I'll ignore "should" feedback for the time being and leave it as technical debt for later

Reply:
Yes, I would not have said anything if you didn't mention your insecurity. :)